conf/default.yaml

```diff
@@ -12,4 +12,4 @@ model_checkpoint:
 defaults:
   - _self_
   - dataset: PB_WallSuperHeat
-  - experiment: temp_unet2d
```

Comment:
sorry could you change this to `paper/unet_arena/pb_temp`
(I just want it to default to the simplest model used in the paper.)